cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Did you leave this in here for me? Or did you mean to add this to the gitignore?

Reply:
The is boilerplate code that comes when you install cypress